### DISCUSSION TITLE
Using setValue and passing in a falsy value  (0) is not working.

### DISCUSSION THREAD

**Discussion by Louis345** [react-hook-form/react-hook-form#8345] [Q&A]
I have a weird issue when I enter 0 as a value it's not registering or saving. I found a hack to solve this issue:
```setValue(`${rowId}.${id}`, value === 0 ? '0' : value);```
This could potentially introduce bugs and I was wondering what the proper solution would be? 

This is how I initialized useForm
```
useForm<AllSavedGoalType>({
    mode: 'onChange',
    defaultValues,
    resolver: yupResolver(schema),
  });
```
I search the docs and I couldn't find any configurations to solve my use-case. Any help would be greatly appreciated.

**Comment by Louis345** [comment]
this was an issue on my codebase,

**Reply by ritikbanger** [reply]
Please share a codesandbox or more information.

**Reply by Louis345** [reply]
It had nothing to do with the library it was an issue with my codebase. How do I close this issue?

**Reply by Moshyfawn** [reply]
It's not an issue; just a discussion. Don't worry about closing it ;)

If you have a clear answer to you question or someone answers your question in these, make sure to mark it as the answer by clicking this
![image](https://user-images.githubusercontent.com/16290753/168490718-4a934cc8-db6f-4a19-ad76-4a7be997f607.png)
